README.md

```diff
@@ -1 +1,86 @@
-# wolmo-utils-ios
\ No newline at end of file
+## WolMo - Utils iOS
+
+WolMo - Utils iOS is a framework which provides specific utilities for iOS used at [Wolox](http://www.wolox.com.ar/).
+
+## Table of Contents
+
+  * [Installation](#installation)
+    * [Carthage](#carthage)
+  * [Usage](#usage)
+    * [ImageFetcher](#image-fetcher)
+    * [ImagePickerService](#image-picker-service)
+  * [Contributing](#contributing)
+  * [About](#about)
+  * [License](#license)
+
+## Installation
+
+### Carthage
+
+[Carthage](https://github.com/Carthage/Carthage) is a dependency manager that builds your dependencies and provides you with binary frameworks.
+
+You can install Carthage with Homebrew using the following command:
+
+```
+brew update
+brew install carthage
+```
+To download wolmo-utils-iOS, add this to your Cartfile:
+```
+github "Wolox/wolmo-utils-ios" "master"
+```
+
+## Usage
+
+Right now the framework has 2 utilities: [ImageFetcher](#ImageFetcher) and [ImagePickerService](#ImagePickerService).
+
+### Image Fetcher
```

Comment:
Just checking, is this documented also in the class itself? If there is no documentation, you could copy and paste since we are at it.